cfme/infrastructure/config_management.py

```diff
@@ -122,10 +122,6 @@ class ConfigManagementAddEntities(View):
     add = Button('Add')
     cancel = Button('Cancel')
 
-    # element attributes changed from id to class in upstream-fine+, capture both with locator
-    flash = FlashMessages('.//div[@id="flash_msg_div"]'
```

Comment:
These are defined under `entities` in the actual view, so there are calls to view.entities.flash.

https://github.com/psav/cfme_tests/blob/86a2aa54102ec46dc9d437d7f710e18511647ae2/cfme/infrastructure/config_management.py#L287

The ConfigManagementView inherits BaseLoggedInPage, so I think the calls to `view.entities.flash` just need to be changed to `view.flash`

Reply:
** Fixed by updates below to `view.entities.flash` **